yt/frontends/sdf/tests/test_outputs.py

```diff
@@ -40,7 +41,7 @@ def internet_on():
 def test_scivis():
     if not internet_on():
```

Comment:
the internet_on function has a call to the `scivis_data`. If this PR switches to the mirror should that function be updated to the mirror?

Reply:
good catch! Thanks!